packages/dapp/src/components/TokenDapp.tsx

```diff
@@ -415,6 +438,37 @@ export const TokenDapp: FC<{
           />
         </form>
       </div>
+      <div className="columns">
+        <form onSubmit={handleSignUnsignedTxSubmit}>
+          <h2 className={styles.title}>Sign Unsigned Tx</h2>
+
+          <div className="columns">
+            <label htmlFor="short-text">Unsigned Tx</label>
```

Comment:
Seems that when input is not in Hex format, the dApp will go back to show the Connect Bottom.

Reply:
Fixed. I removed the check for unsigned tx from the dApp, so now it will receive the error from the extension wallet directly.